corehq/apps/reports/standard/monitoring.py

```diff
@@ -281,6 +295,13 @@ def es_queryset(self, users_by_id):
         end_date = ServerTime(self.utc_now).phone_time(self.timezone).done()
         milestone_start = ServerTime(self.utc_now - self.milestone).phone_time(self.timezone).done()
 
+        touched_total_aggregation = FilterAggregation(
+            'touched_total',
+            filters.AND(
+                filters.date_range('modified_on', gte=milestone_start, lt=end_date),
+            )
+        )
```

Comment:
I don't think you need this since the date range for each landmark is a subset of this date range. you should be able to do this:
```python
def total_touched_count(self):
    return 0 if not self.bucket else self.bucket.doc_count
```